src/config.py

```diff
@@ -86,14 +91,25 @@ def zookeeper_connected(self) -> bool:
         return False
 
     @property
-    def extra_args(self) -> List[str]:
```

Comment:
Do these need to be separate for auth? Could we treat them as a single 'JAVA_ARGS' property that we blanket apply to everything? It's all the same runtime args in the end no?

Reply:
I actually had also thought about this too, but I eventually did not feel to raise the comment. Since @marcoppenheimer raised it, I believe it makes sense to me to address this and I endorse his point

Reply:
@zmraul all right! Thanks for clarification! This is resolved IMHO

Reply:
The exporter variable is only needed when the snap is going to start, whereas the auth part is needed always I think. Maybe auth can stay as part of `extra_args` and the exporter can be added just at start moment.